2020-02-Spring/3_trees/README.md

```diff
@@ -70,21 +70,91 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Recover Binary Search Tree
 
-Source: TODO :bug:
+[LeetCode](https://leetcode.com/problems/recover-binary-search-tree/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Two elements of a binary search tree (BST) are swapped by mistake.
```

Comment:
For the Example Input; Output below, it would be clearer to explain which elements were accidentally swapped and or which elements should be swapped back or restored.

Reply:
Good point!

Reply:
Actually I think telling you what nodes to swap might be too much of a hint for how you should approach the problem.